src/site/content/en/metrics/custom-metrics/index.md

```diff
@@ -277,6 +270,69 @@ try {
   attribute will be ignored.
 {% endAside %}
 
+### Event Timing API
+
+The [First Input Delay (FID)](/fid/) metric measures the time from when a user
+first interacts with a page to the time when the browser is actually able to
+begin processing event handlers in response to that interaction. However, in
+some cases it may also be useful to measure the event processing time itself as
+well as the time until the next frame can be rendered.
+
+This is possible with the [Event Timing
+API](https://wicg.github.io/event-timing/) (which is used to measure FID) as it
+exposes a number of timestamps in the event lifecycle, including:
+
+- [`startTime`](https://w3c.github.io/performance-timeline/#dom-performanceentry-starttime):
+  the time when the browser receives the event.
+- [`processingStart`](https://wicg.github.io/event-timing/#dom-performanceeventtiming-processingstart):
+  the time when the browser is able to beginning processing event handlers for
+  the event.
+- [`processingEnd`](https://wicg.github.io/event-timing/#dom-performanceeventtiming-processingend):
+  time when the browser finishing executing all synchronous code initiated from
+  event handlers for this event.
+- [`duration`](https://wicg.github.io/event-timing/#dom-performanceeventtiming-processingstart):
+  the time between when the browser receives the event until it's able to paint
```

Comment:
I think this is not sufficiently clear. The 'init' time is clear but the 'final' one is the next paint after event handlers. Presumably a browser can paint a next frame after receiving an input but without having handled the input.

Reply:
Done.